README.md

```diff
@@ -31,35 +31,51 @@ Then on one computer listen for connections
 // create a server to listen for secure connections
 const server = node.createServer()
 
-server.on('connection', function (noiseSocket) {
-  // noiseSocket is E2E between you and the other peer
-  // pipe it somewhere like any duplex stream
+start()
```

Comment:
understand why, but let's not wrap in an async start function here and instead perhaps make a comment for that :)

Reply:
ok, will edit